test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
If you do not define a port, or set it to 0, Hapi will find an available port. This is really handy when testing (but you have to use tools like `server.inject`, or base HTTP requests off of `server.info.uri`.